File: src/main/java/net/tassia/pancake/event/group/GroupEvent.java
package net.tassia.pancake.event.group;

import net.tassia.pancake.entity.Group;
import net.tassia.pancake.event.PancakeEvent;

public class GroupEvent extends PancakeEvent {

	public final Group group;

	public GroupEvent(Group group) {
		this.group = group;
	}

}

File: pancake/core/src/commonMain/kotlin/net/tassia/pancake/event/generic/GroupEvent.kt
package net.tassia.pancake.event.generic

import net.tassia.pancake.entity.Group

/**
 * Declares that a given event involves a [Group].
 *
 * @since Pancake 1.0
 * @author Tassilo
 */
interface GroupEvent {

	/**
	 * The group.
	 */
	val group: Group

}
